tests/extract/conftest.py

```diff
@@ -181,11 +174,13 @@ def env_setup():
     ]
     for a in attempt:
         try:
-            MySQL(host=a)
-            os.environ["DATABASE_URL"] = a
+            with MySQL(host=a):
+                os.environ["DATABASE_URL"] = a
         except Exception:
             pass
 
+    assert os.environ["DATABASE_URL"]
+
```

Comment:
Nice to have this fail here, instead of failing in the middle of some other test.